### PR TITLE
disable smart-update for individual & fix custom interval

### LINKED ISSUE #306: Add N/A on custom update frequency
### Describe your suggested feature

There should be a more customized option for when you don't want a specific entry to update, so please add an N/A on the custom update frequency

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.11.2](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by kana-shii**
eventually closes #306 when I figure this out
(it's so hard)
what is working:
N/A is being displayed as the second option and it looks like everything that has the N/A isn't updating while if it has default or any other thing it does update
what isn't working:
I tried making it so when an user chooses the N/A option, it will make the MangaInfoHeader appear as N/A as well on the hourglass. It kinda failed...? everything started appearing as N/A after that.

### CODE REVIEW

File: app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt [hunk 1]
```diff
@@ -117,17 +117,19 @@ fun SetIntervalDialog(
                         contentAlignment = Alignment.Center,
                     ) {
                         val size = DpSize(width = maxWidth / 2, height = 128.dp)
-                        val items = (0..FetchInterval.MAX_INTERVAL)
+                        val items = (-1..FetchInterval.MAX_INTERVAL)
```
Comment (cuong-tran) [inline app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt]: You can create a list with -1 at the end like this: `(0..FetchInterval.MAX_INTERVAL) + listOf(-1))`

Comment (cuong-tran) [reply app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt]: Another option is add a LongClick, when user long click the timeglass then set update interval to -1. Remember to show a toast message.

**Comment by cuong-tran** [comment]
1. If both option "Predict" & "Skip Completed" are on, it still allows user to set a date on Completed entry.
 - Please test if when user sets a date to Completed entry, will it update?
 - Issue: When set a date on Completed entry, it seems the hourglass is light up but still saying N/A.
2. If only "Predict" is on and "Skip Compelted" is off.
- Light up the hourglass when it is set to N/A. Also change the hourglass to hourglass-disabled

**Comment by cuong-tran** [comment]
There's a case when manga has no chapter, the hourglass is lighting up. Not sure it should be like that?

File: app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt (hunk 1 above)
Comment (kana-shii) [reply app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt]: > You can create a list with -1 at the end like this: `(0..FetchInterval.MAX_INTERVAL) + listOf(-1))`

I'll try that out

> Another option is add a LongClick, when user long click the timeglass then set update interval to -1. Remember to show a toast message.

This is something that would be really cool to add tbh. I'll look into it

**Comment by kana-shii** [comment]
> 1. If both option "Predict" & "Skip Completed" are on, it still allows user to set a date on Completed entry.
> 
> * Please test if when user sets a date to Completed entry, will it update?
> * Issue: When set a date on Completed entry, it seems the hourglass is light up but still saying N/A.

This is one of the issues I've put on the first comment of this commit. I'm trying to fix that still

> 2. If only "Predict" is on and "Skip Compelted" is off.
> 
> * Light up the hourglass when it is set to N/A. Also change the hourglass to hourglass-disabled

I'll do that, but I still need to figure out how to make the N/A not appear on everything tbh

> Also, when user sets a custom date, it's only showing `Soon`. I think this should be changed to the actual remaining days.

I'll also try to fix this

This PR is much harder than I thought lmao but I'll still try working on it

**Comment by kana-shii** [comment]
> There's a case when manga has no chapter, the hourglass is lighting up. Not sure it should be like that?

It's always been like that though

File: app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt (hunk 1 above)
Comment (kana-shii) [reply app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt]: If I do that, it starts having other errors through the whole project

**Pull request merged**